### PR TITLE
[TR] Translation of README-TR.md

### PR DESCRIPTION
**Pull request by berabulut**
I completed translation of [README.md](https://github.com/Atcold/pytorch-Deep-Learning/blob/master/README.md) to Turkish  and created [tr] folder.

### CODE REVIEW

**Comment by Atcold** [comment]
Wow, impressive. Please, update all the other `README.md` to reflect the new added language.

**Comment by berabulut** [comment]
I added them too.

**Comment by Atcold** [comment]
Perfect! Thanks so much!

File: docs/tr/README-TR.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+<!-- Deep Learning (with PyTorch)-->
+# Derin Öğrenme (PyTorch ile) [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/Atcold/pytorch-Deep-Learning/master)
+
+<!-- This notebook repository now has a [companion website](https://atcold.github.io/pytorch-Deep-Learning/), where all the course material can be found in video and textual format.
+-->
+
+Bu kurs deposunun materyallerini yazılı ve video halinde bulabileceğiniz [ilgili site](https://atcold.github.io/pytorch-Deep-Learning/).
```
Comment (Atcold) [inline docs/tr/README-TR.md:7]: Here the link should point to the Turkish version, not the English one. Check the other translations. But yeah, at the moment there is no Turkish website yet. Still, let's fix this as a PR I will accept when there will be an `index-TR.md`, okay?

**Pull request merged**